src/js/rollup.config.js

```diff
@@ -33,7 +35,12 @@ function config({ input, output, name, format, minify }) {
       nodeResolve(),
       minify
         ? terser({
-            compress: true,
+            compress: {
+              dead_code: true,
+              global_defs: {
+                DEBUG,
+              },
+            },
```

Comment:
Do we maybe not want to compress in debug mode? Or have an environment variable to disable compression?

Reply:
Good point, I should also disable the compression